doc/workers.rst

```diff
@@ -105,45 +142,87 @@ A very short how-to for creating such an AMI manually:
         ~ $ conda env update --name base --file $ami_environment
         ~ $ conda list
 
-  - Get the data::
+  - Get the data and copy public data to ``ramp-kits/iris/data``::
 
         ~ $ data_dir="$kit_dir/data"
         ~ $ rm -rf $data_dir && mkdir $data_dir
         ~ $ git clone https://github.com/ramp-data/iris ramp-data/iris
         ~ $ cd ramp-data/iris/
         ~ $ python prepare_data.py
-        ~ $ cd ..
-
-    TODO: figure out this data (in ramp-kits/data or in ramp-data?)
+        ~ $ cp data/public/* ~/ramp-kits/iris/data/
 
   - Test the kit::
 
-        ~ $ cd ramp-kits/iris
+        ~ $ cd ~/ramp-kits/iris
         ~ $ ramp-test
 
-- Save the instance as an AMI: from the instance -> Actions -> Image -> Create image
+Next, save the instance as an AMI. Starting from the instance tab:
+Actions -> Image -> Create image. See `Create an AMI from an Amazon EC2
+instance <https://docs.aws.amazon.com/toolkit-for-visual-studio/latest/user-guide/tkv-create-ami-from-instance.html>`_
+for more details.
+
+Event configuration
+^^^^^^^^^^^^^^^
 
-- Create an event config.yml, which should look something like::
+Create an event config.yml (see :ref:`deploy-ramp-event`) and update the
+'worker' section, which should look something like::
 
       ramp:
           problem_name: iris
           event_name: iris_ramp_aws_test
-          event_title: iris_ramp_aws_test
+          event_title: "iris ramp aws test"
           event_is_public: true
-          ...
       worker:
           worker_type: aws
           access_key_id: <aws_access_key_id for boto3 Session>
           secret_access_key: <aws_secret_access_key for boto3 Session>
           region_name: us-west-2 # oregon
           ami_image_name: <name of the AMI set up for this event>
-          ami_user_name : ubuntu
-          instance_type : t2.micro
+          ami_user_name: ubuntu
+          instance_type: t2.micro
           key_name: <user that can ssh connect with the created AWS instance>
```

Comment:
Again I think this is the name of the key not the name of the user ?

Reply:
I'm not 100% sure that the key_name is the name of the ssh key, because my text ultimately failed. Did you double check this on your machine...?